src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -114,6 +126,11 @@ public void addPerson(Person person) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void addPersonAtIndex(Person person, Index index) {
+            throw new AssertionError(("This method should not be called."));
```

Comment:
maybe can extract out as constant and place in `TestUtil` class cus i think got quite a lot of tests that uses this message.